docs/config.rst

```diff
@@ -438,6 +446,17 @@ Features
 
    .. versionadded:: 2.0
 
+.. attribute:: LP_ENABLE_DISPLAY
+   :type: bool
+   :value: 1
+
+   Displays a green ``@`` if the connection has X11 support;
+   a yellow one if not.
```

Comment:
This is correct for the default theme, but not for other themes. Should probably be reworded.

```suggestion
   Detect if the connection has X11 support.
   
   In the default them, display a green ``@`` if it does; a yellow one if not.
```